### DISCUSSION TITLE
PBP preconditioner

### DISCUSSION THREAD

**Discussion by nanguaxiaofendui** [idaholab/moose#23668] [Q&A General] [answered]
Hello all,
What parameters can be set for preprocessor when using PBP? Where else do I need to find all the available parameters? thank you.

**Comment by GiudGiud** [comment, marked as answer]
Hello

list of parameters at the bottom of this page
https://mooseframework.inl.gov/source/preconditioners/PhysicsBasedPreconditioner.html

Guillaume

**Reply by nanguaxiaofendui** [reply]
thank you

**Reply by GiudGiud** [reply]
it s not super-finished in terms of documentation but examples (at bottom of page) will help too

**Reply by nanguaxiaofendui** [reply]
thanks
